argoverse/evaluation/eval_tracking.py

```diff
@@ -304,9 +277,40 @@ def eval_tracks(
     num_miss = summary["num_misses"][0]
     num_switch = summary["num_switches"][0]
     num_frag = summary["num_fragmentations"][0]
+    write_summary_to_file(
+        acc_c,
+        acc_o,
+        ID_gt_all,
+        path_tracker_output,
+        num_frames,
+        mota,
+        motp_c,
+        idf1,
+        most_track,
+        most_lost,
+        num_fp,
+        num_miss,
+        num_switch,
+        num_frag,
+    )
 
-    acc_c.events.loc[acc_c.events.Type != "RAW", "D"] = acc_i.events.loc[acc_c.events.Type != "RAW", "D"]
 
+def write_summary_to_file(
+    acc_c,
+    acc_o,
+    ID_gt_all: List[str],
```

Comment:
Does this actually _have_ to be `List` or can it be `Collection` or `Iterable` or `Sequence`?